.github/workflows/haskell.yml

```
@@ -5,6 +5,9 @@ on:
     branches: [ "master" ]
   pull_request:
     branches: [ "**" ]
+  schedule:
+    # Every night at 02:00 UTC
+    - cron: '0 2 * * *'
```

We figured 6am UTC is better for everyone on the team, since no-one is around during that time. See https://github.com/input-output-hk/cardano-ledger/issues/3246
```suggestion
    # Every night at 06:00 UTC
    - cron: '0 6 * * *'
```

---

Ok., but that would make them matinal builds instead of nightly :laughing:

---

It would for London :stuck_out_tongue_winking_eye:

---

let's call them happy hour tests. it's five o'clock somewhere! :beers:

---

:rofl: That's awesome